platform/core.windows/src/org/netbeans/core/windows/services/NbPresenter.java

```diff
@@ -503,20 +500,12 @@ public int getMaxCharactersPerLineCount() {
             );
         }
 
-        if (UIManager.getLookAndFeel().getClass() == MetalLookAndFeel.class ||
-            UIManager.getLookAndFeel().getClass() == BasicLookAndFeel.class) {
-            optionPane.setUI(new javax.swing.plaf.basic.BasicOptionPaneUI() {
-                @Override
-                public Dimension getMinimumOptionPaneSize() {
-                    if (minimumSize == null) {
-                        //minimumSize = UIManager.getDimension("OptionPane.minimumSize");
-                        // this is called before defaults initialized?!!!
-                        return new Dimension(MinimumWidth, 50);
-                    }
-                    return new Dimension(minimumSize.width, 50);
-                }
-            });
-        }
+        // TODO move to a better place, only for testing
```

Comment:
Perhaps the TODO comment can go away, otherwise good!

Reply:
will do, I wasn't sure if we should leave the
```
UIManager.getDimension("OptionPane.minimumSize").setSize(BasicOptionPaneUI.MinimumWidth, 38);
```
there or move it to some other place. This would keep overwriting the value and therefore not allow any customization.

Reply:
I can defer to your opinion here. Customization is only necessary if you think someone will need to customize it.

My own NetBeans Platform app has been using a patched version of this class for 2 years (the one from https://github.com/apache/netbeans/pull/5335 ), and if the official version doesn't end up working for me, I'll just keep using my old patch.

Reply:
ok